tcs-persistence/src/test/java/com/transformuk/hee/tis/tcs/service/repository/PostRepositoryTest.java

```diff
@@ -1,5 +1,8 @@
 package com.transformuk.hee.tis.tcs.service.repository;
 
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+import static org.junit.jupiter.api.Assertions.assertNull;
```

Comment:
This assertNull is from JUnit 5, but the whole tests are on JUnit 4.  Just wondering if you want to make them unified.

Reply:
Yeah. Very good observation.